example/windows/win32_window.cc

```diff
@@ -109,7 +122,7 @@ Win32Window::MessageHandler(HWND hwnd, UINT const message, WPARAM const wparam,
       SetWindowPos(hwnd, nullptr, lprcNewScale->left, lprcNewScale->top,
                    newWidth, newHeight, SWP_NOZORDER | SWP_NOACTIVATE);
 
-      return 0;
+      break;
```

Comment:
Why don't we want to indicate that we've handled the message?

Reply:
The examples that I've seen still call the `DefWindowProc`. Is there a reason to stop the propagation?